app/controllers/api/v1/regions_controller.rb

```diff
@@ -1,5 +1,11 @@
 class Api::V1::RegionsController < ApplicationController
-    def index
-        render json: Region.all
-    end
+  def index
+    render json: Region.all.order('name')
+  end
+
+  def show
+    @region = Region.find(params[:id])
+    @flavors = @region.flavors
+    render json: @flavors.order('name')
+  end
```

Comment:
Does a region show page not display the name of the region? It looks like its just returning the associated flavors currently.  We can make use of `serializers` and then we can simply return,
```ruby
def show
  render json: Region.find(params[:id])
end
```

and on the serializer we just have to call `has_many :flavors`

Reply:
Added serializers and updated related files